src/components/UI/Tables/GenericTable/index.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  TableContainer,
+  Paper,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+} from "@mui/material";
+
+export interface IGenericTableColumn {
+  key?: string;
```

Comment:
The key is required in the list but prefers to use id, field, name, header base on the column

Reply:
Fixed